package.json

```diff
@@ -20,9 +20,9 @@
   "license": "MIT",
   "dependencies": {
     "@auth0/thumbprint": "0.0.6",
-    "ejs": "2.5.5",
+    "ejs": "^3.1.9",
```

Comment:
The reasons for major bump are [documented here](https://github.com/mde/ejs/commit/4836295f30beac1698ff6508da93682ee7c4f812):

```
* Removed require.extensions (@mde)
* Removed legacy preprocessor include (@mde)
* Removed support for EOL Nodes 4 and 6 (@mde)
```

We aren't using require.extensions.
We aren't using `include` in our templates.
Support for Node < 12 was dropped in version 7 of node-wsfed.